### ISSUE TITLE
Missing Accessibility Toolbar in projects with no iDevices.

### ISSUE THREAD

**Issue by ignaciogros** [exelearning/exelearning#1437]
Export or preview a project with no iDevices.

The Accessibility Toolbar is missing.
